CHANGELOG.md

```diff
@@ -4,6 +4,42 @@ Notable changes to Draft.js will be documented in this file.
 
 Changes to `src` are live in production on facebook.com at the time of release.
 
+## 0.10.5 (January 19th, 2018)
+
+### Added
+* Add support for `ariaDescribedBy` prop, for better a11y. (Suraj Karnati in
+  [a6af3e15](https://github.com/facebook/draft-js/commit/a6af3e15120e74c8797c5670f5bb63cb45c49a32))
+* Add support for `ariaLabelledBy` prop, for better a11y. ([@jackyho112](https://github.com/jackyho112)
+  in [#1519](https://github.com/facebook/draft-js/pull/1519))
+
+### Changed
+* Cause editor to break out of code block when user enters two blank lines. (Hanzhi Zhang
+  in [548fd5d1](https://github.com/facebook/draft-js/commit/548fd5d1b1c31b7b4c79cd70b101fae69d522b3f))
+
+### Fixed
+* Preserve list indentation when copying and pasting from one Draft.js editor
+  into another. ([@GordyD](https://github.com/GordyD) in [#1605](https://github.com/facebook/draft-js/pull/1605))
+* Fix `cannot read property 'update' of undefined` error that was thrown when
+  typing same character into selection that starts with that character. ([@existentialism](https://github.com/existentialism) in
+  [#1512](https://github.com/facebook/draft-js/pull/1512))
+* Fix `encodeRawBlocks` to handle non-contiguous entities. Entities should
```

Comment:
nit: Entities should always be `contigious -> contiguous`...

Reply:
Thanks! Will fix in a follow-up - don't want to make CI restart, it is already taking forever-and-a-half.

Reply:
Sounds good!